tests/unit/helpers/RemoteControlTest.php

```diff
@@ -176,6 +176,89 @@ public function testAddResponse()
         self::$testSurvey = null;
     }
 
+
+    /**
+     * Test the add_response API call.
+     */
+    public function testDeleteResponse()
+    {
+        \Yii::import('application.helpers.remotecontrol.remotecontrol_handle', true);
+        \Yii::import('application.helpers.viewHelper', true);
+        \Yii::import('application.libraries.BigData', true);
+        $dbo = \Yii::app()->getDb();
+
+        // Make sure the Authdb is in database (might not be the case if no browser login attempt has been made).
+        $plugin = \Plugin::model()->findByAttributes(array('name'=>'Authdb'));
+        if (!$plugin) {
+            $plugin = new \Plugin();
+            $plugin->name = 'Authdb';
+            $plugin->active = 1;
+            $plugin->save();
+        } else {
+            $plugin->active = 1;
+            $plugin->save();
+        }
+        App()->getPluginManager()->loadPlugin('Authdb', $plugin->id);
+        // Clear login attempts.
+        $query = sprintf('DELETE FROM {{failed_login_attempts}}');
+        $dbo->createCommand($query)->execute();
+
+        // Import survey
+        $filename = self::$surveysFolder . '/limesurvey_survey_666368.lss';
```

Comment:
Maybe more simple to import a LSA with two response ?

Then no need to test add response for a second time :) : one function => one test

Reply:
Can you point me to a data survey which has 2 responses already ? Does this exists ? Should I create one ?

Reply:
Just import 666368 survey, launch it manually in your instance, export as Survey archive limesurvey_survey_add_delete_RPC_function_develop.lsa, and put the good name here.

If you want to use an existing one : https://github.com/LimeSurvey/LimeSurvey/tree/master/tests/data/surveys survey_archive_statCountFunctionsTest.lsa have some reponse.